tests/test_api_lib_policy.py

```diff
@@ -1523,6 +1523,31 @@ def test_24_push_wait_policy(self):
 
         delete_policy("push1")
 
+    def test_24b_push_disable_wait_policy(self):
+        # We send a fake push_wait that is not in the policies
+        builder = EnvironBuilder(method='POST',
```

Comment:
Do we need to build a complete environment? If we call the function directly, there won't be any request handling, correct?
So here we can just test if the function does the right thing (adjusting the `all_data` entry).

Reply:
That is true :) Fixed in 3769619.